spec/controllers/introduce_yourself_controller_spec.rb

```diff
@@ -38,6 +38,26 @@
   end
 
   describe "#update" do
+    context "name contains leading or trailing whitespace" do
+      it "strips whitespace" do
+        params = {
+          step: {
+            first_name: " Space ",
+            last_name: " Case ",
+            "birthday(3i)" => "31",
+            "birthday(2i)" => "1",
+            "birthday(1i)" => "1950",
+          },
+        }
+
+        put :update, params: params
+        member.reload
+
+        expect(member.first_name).to eq "Space"
+        expect(member.last_name).to eq "Case"
+      end
+    end
```

Comment:
for using `.try` above, would it be worthwhile to add assertions about nil inputs?